src/app/messages/en/messages.json

```diff
@@ -93,9 +93,12 @@
   "i18n:page-not-found:heading": "Uh oh, we can't find the page you requested.",
   "i18n:page-not-found:text-before-button": "Try another page in the",
   "i18n:practice-questions:popup:close": "Click to close Practice Questions modal",
+  "i18n:practice-questions:popup:continue": "Continue",
   "i18n:practice-questions:popup:heading": "Practice Questions",
   "i18n:practice-questions:popup:questions": "Questions",
   "i18n:practice-questions:popup:read": "Read",
+  "i18n:practice-questions:popup:empty:message": "There are no questions for this chapter/section.",
```

Comment:
alphabetic order

Reply:
I think we are also sorting them by their "depth" - indicated by `:`?
Similarly to:
```
"i18n:studyguides:popup:filters:yellow": "Key Terms",
"i18n:studyguides:popup:filters:filter-by:aria-label":
```

Reply:
Or shouldn't we?

Reply:
in this case they're also wrong, `"i18n:studyguides:popup:filters:filter-by:aria-label":` should be before because `filter-by` goes before `yellow`